### ISSUE TITLE
Ctrl-Shift-i not working in Google Chrome 45.0.2454.101 (64-bit)

### ISSUE THREAD

**Issue by 11111000000** [gaearon/redux-devtools-dock-monitor#14]
Reason:

``` javascript
DockMonitor.prototype.handleKeyDown = function handleKeyDown(e) {
    if (!e.ctrlKey) {
      return;
    }
    e.preventDefault();

```

with commented `e.preventDefault();` all works as expected

**Comment by 11111000000** [comment]
ah, and I see this on beta-3 ... there is update?

**Comment by cbeihl** [comment]
This appears to be fixed by https://github.com/gaearon/redux-devtools-dock-monitor/commit/1e6980ae181bb773b0d5451d5316ad1b011789ec on Oct 18.  When will v1.0.0-beta-4 be released ?

**Comment by gaearon** [comment]
The fix is released in 1.0.1, requires DevTools 3.0 stable.
